src/components/dialogmote/avlys/AvlysDialogmoteSkjema.tsx

```diff
@@ -45,6 +59,13 @@ const SendButton = styled(TrackedHovedknapp)`
   margin-right: 0.5rem;
 `;
 
+const AvlysningAlertStripe = styled(AlertStripeAdvarsel)`
+  margin-bottom: 4em;
+  .alertstripe__tekst {
```

Comment:
Hm, ikke så gode erfaringer fra tidligere av med override designsystem CSS. Plutselig endrer de sin navngivning, eller gjør andre endringer som gjør at override feiler.

Er det mulig å bruke standard styling, eventuelt lage en PR til designsystem der vi kan slenge med en prop for å få max width?

Reply:
Sant. Kan undersøke det nærmere 🧐

Reply:
I følge designsystem-gjengen bør ikke tekstlinje være lenger enn 80 bokstaver så man ønsker derfor ikke eksponere denne som prop. Har ikke funnet noen annen måte å overstyre denne på da enn å sett max-width på alertstripe__tekst.

Reply:
Skjønner. Men burde vi også da holde oss til den anbefalingen?

Reply:
Det kan hende :) Men mulig det er ment for eksterne flater... kan ta en runde med Espen om det.